FrontEnd/src/components/HeaderFooter/header/navbar/Profile.jsx

```diff
@@ -1,12 +1,14 @@
-import css from "./Profile.module.css"
-import avatar_image from "./Avatar.png"
+import css from "./Profile.module.css";
+import avatar_image from "./Avatar.png";
+import { Link } from 'react-router-dom';
```

Comment:
👍

Reply:
I don’t know why I changed something here, but  in a new commit I put the imports in alphabetical order as it was before